Reactions/ReactorArkode.cpp

```diff
@@ -110,8 +110,6 @@ ReactorArkode::init(int reactor_type, int /*ncells*/)
   } else {
     amrex::Print() << "ARK Step:" << std::endl;
   }
-  amrex::Print() << "  Setting tolerances rtol = " << relTol
```

Comment:
did you mean to remove these? I am fine either way.

Reply:
yes, this was intended. The tolerances are printed out again through the set_sundials_tols function anyway when verbose is on.